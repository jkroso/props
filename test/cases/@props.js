@name == add(@first, @last) == add(first_name, last_name)