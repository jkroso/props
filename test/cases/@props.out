add first_name last_name